### DISCUSSION TITLE
How to find the references for the algorithms or datasets, e.g. "colour.SDS_LIGHT_SOURCES" attribute?

### DISCUSSION THREAD

**Discussion by hminle** [colour-science/colour#1031] [Q&A] [answered]
Hi,

I would like to use the SDS Light sources and cite them in my research works. But unfortunately I cannot find citation and related papers for these sources

https://colour.readthedocs.io/en/develop/generated/colour.SDS_LIGHT_SOURCES.html

Could you help me?

Thanks

**Comment by thomasmansencal** [comment, marked as answer]
Hi @hminle,

The references are available at the bottom of the page you linked or at the top of the module too: https://github.com/colour-science/colour/blob/develop/colour/colorimetry/datasets/light_sources/sds.py

Cheers,

Thomas

**Reply by hminle** [reply]
Hi @KelSolaar ,

Yeah I see it. But I cannot search them on Google Scholar. It seems that they are not official citations, right?

Thanks,
Hoang

**Reply by thomasmansencal** [reply]
Unfortunately, not everything has a DOI or journal reference.

**Reply by hminle** [reply]
Thank Thomas.
